api/v1alpha1/fenceagentsremediationtemplate_webhook.go

```diff
@@ -37,8 +39,24 @@ func (r *FenceAgentsRemediationTemplate) SetupWebhookWithManager(mgr ctrl.Manage
 
 // TODO(user): EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 
+// +kubebuilder:webhook:path=/mutate-fence-agents-remediation-medik8s-io-v1alpha1-fenceagentsremediationtemplate,mutating=true,failurePolicy=fail,sideEffects=None,groups=fence-agents-remediation.medik8s.io,resources=fenceagentsremediationtemplates,verbs=create;update,versions=v1alpha1,name=mfenceagentsremediationtemplate.kb.io,admissionReviewVersions=v1
+
+var _ webhook.Defaulter = &FenceAgentsRemediationTemplate{}
+
+// Default implements webhook.Defaulter so a webhook will be registered for the type
+func (farTemplate *FenceAgentsRemediationTemplate) Default() {
+	webhookFARTemplateLog.Info("default", "name", farTemplate.Name)
+	if farTemplate.GetAnnotations() == nil {
+		farTemplate.Annotations = make(map[string]string)
+	}
+	if _, isSameKindSupported := farTemplate.GetAnnotations()[commonAnnotations.MultipleTemplatesSupportedAnnotation]; !isSameKindSupported {
+		farTemplate.Annotations[commonAnnotations.MultipleTemplatesSupportedAnnotation] = "true"
+	}
```

Comment:
It might just be me, but reading the condition `!isSameKindSupported` makes me think that the condition is _not
supported_, instead than just the annotation unset. It might longer, but what do you think about
`isSameKindAnnotationSet` instead?

Reply:
Good point, I agree :+1: